training-courses/valuenotvulns/bheu23.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Building a High-Value AppSec Scanning Programme
+subtitle: Understanding how to get better value from SAST, DAST and SCA tools.
+layout: service
+hero_height: is-small
+backlink: /training
+backname: Training
+menubar: valuenotvulns_menu
+call_link: NONE
+---
+
+###### Thanks for your interest in “Building a High-Value AppSec Scanning Programme” at Black Hat EU 2023! 
+
+You can register directly via [this link](https://www.blackhat.com/eu-23/training/schedule/index.html#building-a-high-value-appsec-scanning-programme-sca-sast-dast-and-more-virtual-33767) or click around for more information about the course. (Keep this page open separately as it is not linked elsewhere on the site!)
```

Comment:
Sorry @joshbouncesecurity I made comments and forgot to submit them...

Reply:
So it isn't linked anywhere so I figured we would just leave it in case we want it some other time, or do you prefer that it is not there at all? @avidouglen

Reply:
Ah fair enough, if we're not actively using it doesnt really matter... Doesnt look great though, so for next time :-)